.docker/Dockerfile.test

```diff
@@ -9,3 +9,14 @@ RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSI
     && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
 COPY --from=cli /app /app
+
+COPY tests /app/tests
```

Comment:
This is better mounted as a volume, to avoid re-building environment to refresh any new or changed tests.
@srowlands 
In docker-compose.yml:
```
- ./tests/behat:/app/web/tests/behat:${VOLUME_FLAGS:-delegated}
- ./tests/phpunit:/app/web/tests/phpunit:${VOLUME_FLAGS:-delegated}
```